### ISSUE TITLE
### <span aria-hidden="true">❌</span> Deploy Preview for *stackblitz-docs* failed. **[Why did it fail? →](https://app.netlify.com/projects/stackblitz-docs/deploys/6954b35a0b2d5100089614f4?deploy_diagnostics=true)**

### ISSUE THREAD

**Issue by ahamed00879-sketch** [stackblitz/docs#257]
### <span aria-hidden="true">❌</span> Deploy Preview for *stackblitz-docs* failed. **[Why did it fail? →](https://app.netlify.com/projects/stackblitz-docs/deploys/6954b35a0b2d5100089614f4?deploy_diagnostics=true)**

|  Name | Link |
|:-:|------------------------|
|<span aria-hidden="true">🔨</span> Latest commit | eaf8376a5cd38bd95be98b613425bc8af40813c7 |
|<span aria-hidden="true">🔍</span> Latest deploy log | https://app.netlify.com/projects/stackblitz-docs/deploys/6954b35a0b2d5100089614f4 |

_Originally posted by @netlify[bot] in https://github.com/stackblitz/docs/issues/255#issuecomment-3701470638_

**Comment by ahamed00879-sketch** [comment]
Tgy
